src/Symfony/Component/Form/Tests/Extension/Core/DataTransformer/PercentToLocalizedStringTransformerTest.php

```diff
@@ -144,6 +144,7 @@ public function testDecimalSeparatorMayNotBeDotIfGroupingSeparatorIsDot()
         // Since we test against "de_AT", we need the full implementation
         IntlTestHelper::requireFullIntl($this, '4.8.1.1');
 
+
```

Comment:
an extra line here fixes the issue? would it work if the line was a comment instead?

Reply:
Nope, I just wanted to find out on which branch the AppVeyor build started to fall. Because of the linked PR I thought it should have been this branch. Turns out it wasn't.

Reply:
However, merging #24460 into the `2.7` branch and merge it from there should fully resolve the issue.